src/network/clientpackethandler.cpp

```diff
@@ -1104,6 +1105,11 @@ void Client::handleCommand_HudAdd(NetworkPacket* pkt)
 	}
 	catch(PacketError &e) {}
 
+	try {
+		*pkt >> text2;
+	}
+	catch(PacketError &e) {}
```

Comment:
All the previous try-catch blocks should be merged together actually

Reply:
By <i>all</i> I really meant all of them.
I don't know why the other catches `PacketError`, just make it catch `SerializationError`.